content/api/commands/intercept.md

```diff
@@ -212,9 +214,41 @@ For more guidance around aliasing requests with GraphQL, see [Working with Graph
 
 </Alert>
 
+### Matching `url` and `method` with [RouteMatcher](#routeMatcher-RouteMatcher)
```

Comment:
These "Matching" example should immediately follow the other two "Matching" examples.

Reply:
Yes, they should 😄  That was my intention. Good catch!